src/doc/alerts.xml

```diff
@@ -86,7 +86,8 @@ This list the alerts
 10046	Insecure Component
 10047	HTTPS Content Available via HTTP
 10048	ShellShock
-10049  Image Location Scanner - passive scanner finding
+10049   Image Location Scanner - passive scanner finding
```

Comment:
Indentation changes are better if done in other commit than the actual changes (but I guess a little change like this is just fine). [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines